### DISCUSSION TITLE
DO NOT USE v14.2.8

### DISCUSSION THREAD

**Discussion by ZoranRavic** [vercel/next.js#69769] [App Router]
If you did install next.js v14.2.8 just trust me and roll back to v14.2.7

Multiple issues with server actions were introduced and they seem hard to debug and may not be easy to reproduce.

I did not do QA for the our entire store website after updating next.js, which turned out to be a big mistake as we got a lot of angry customers who quickly encountered issues.

Considering that v14.2.8 had a large changelog compared to other patches, this should not have been a patch release, as patch releases are meant to indicate to developers that not much has changed.

**Comment by Juneezee** [comment]
There are other users experiencing the same issue. There is an open issue for this https://github.com/vercel/next.js/issues/69756

**Reply by ZoranRavic** [reply]
I've seen that issue, but that is not the only error.

For example one odd issue we had is that calling an action might reload page props for no reason.  
(no errors in console, the action returned the correct response)  
These new page props (list of products in our case) are different, thus the UI is effectively reset, which makes a part of the website unusable for users.  
And the only fix is to rollback to v14.2.7.  
The biggest problem with that bug was that it only happened in a use case scenario that we have on production and I couldn't reproduce it in a development/preview environment, not yet at least, I'll try again later.

When I have time I'll try making an example repo of how to reproduce it, if it is new fixed in a future release in the meantime.
